src/components/navBar/index.tsx

```diff
@@ -1,17 +1,106 @@
 import Image from 'next/image';
 import styles from '@/components/navBar/navBar.module.scss';
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import { USER_SELF } from '../constants/url'
+import navLinks from './navlink'
 
 const RDSLogo = '/RDSLogo.png';
+const GitHubLogo = '/gitHubLogo.svg'
+const hamburgerLogo = '/hamburger.svg';
+const DEFAULT_AVATAR = '/dummyProfile.png'
+const SIGN_IN_WITH_GITHUB = 'Sign In With GitHub'
+const SIGN_IN = 'Sign In'
 
-const NavBar = () => (
-  <nav className={styles.navBar}>
-    <a className={styles.logo} href="https://realdevsquad.com"><Image width="45px" height="45px" src={RDSLogo} alt="real-dev squad" /></a>
-    <a href="https://welcome.realdevsquad.com/">Welcome</a>
-    <a href="https://www.realdevsquad.com/events.html">Events</a>
-    <a href="https://members.realdevsquad.com/">Members</a>
-    <a href="https://crypto.realdevsquad.com/">Crypto</a>
-    <a className={styles.active} href="https://status.realdevsquad.com/">Status</a>
-  </nav>
-);
+const NavBar = () => {
+  const [toggle, setToggle] = useState(false);
+  const [userData, setUserData ] = useState({})
+  const [userisLoggedIn, setUserisLoggedIn] = useState(false)
+
+  useEffect(()=>{
+    (async function(){
+      try{
+        const { data, status, statusText, incompleteUserDetails } = await axios.get(USER_SELF,{withCredentials:true})
+        console.log
+        if(status !== 200){
+          throw new Error(`${status} (${statusText})`);
+        }else if(incompleteUserDetails){
+          window.open(
+            'https://my.realdevsquad.com/signup',
+            '_blank',
+            'noopener'
+          );
+        }
+
+        setUserisLoggedIn(true)
+        setUserData({
+          userName: data.username,
+          firstName:data.first_name,
+          profilePicture:data.picture?.url ?? DEFAULT_AVATAR
+        })
+      }catch(e){
+        console.log(e.message)
+      }
+
+    })()
+  },[])
+
+  const UserSignIn=({btnText})=>{
+    return !userisLoggedIn ?  <a
```

Comment:
Underline is visible under Sign In text in mobile view
I think adding `text-decoration: none` to this anchor tag will fix it.

![image](https://user-images.githubusercontent.com/46373689/163728371-908d9c91-2e39-46f9-8242-635d7982520a.png)

Reply:
Fixed